mcs/class/System.IO.Compression/Test/System.IO.Compression/ZipTest.cs

```diff
@@ -507,6 +507,11 @@ class MyFakeStream : FileStream
 			/// Simulate "CanSeek" is false, which is the case when you are retreiving data from web.
 			/// </summary>
 			public override bool CanSeek => false;
+
+			public override long Position {
```

Comment:
Non-seekable streams typically do not support even accessing the position property.